tests/unit/dataactvalidator/test_scripts_loadFSRS.py

```diff
@@ -0,0 +1,73 @@
+from unittest.mock import Mock
+
+import pytest
+
+from dataactcore.models.fsrs import FSRSAward, FSRSSubaward
+from dataactvalidator.scripts import loadFSRS
+from tests.unit.dataactcore.factories import (
+    FSRSAwardFactory, FSRSSubawardFactory)
+
+
+@pytest.yield_fixture()
```

Comment:
Been reading up on pytest fixtures in the docs, and it looks like the `yield_fixture` decorator is deprecated (_i.e._, you don't need it to use `yield` for teardown).

To be clear, I'm reading these docs not to check up on you, but to understand what is happening.